src/templates/xpath_validation_errors.html

```diff
@@ -1,10 +1,7 @@
 <div class="alert alert-danger">
     <h4><i class="fa fa-warning"></i> Validation Failed</h4>
-    <p>Please fix all errors before leaving this page:</p>
-    <%=errors%>
-    <ul>
-    <% _.each(errors, function (error) { %>
-        <li><%=error%></li>
-    <% }); %>
-    </ul>
+    <p>Please fix the error before leaving this page:</p>
```

Comment:
vellum doesn't currently have translations unfortunately. and I don't think we can rely on django to fix this because the templates are compiled into the vellum javascript code